cli/internal/cloudcmd/tfplan.go

```diff
@@ -55,7 +55,7 @@ func plan(
 
 	// If we are planning in a new workspace, we don't want to show a diff
 	if isNewWorkspace {
-		return false, nil
+		return hasDiff, nil
```

Comment:
nit: if the comment still applies it's clearer to just return false no?
```suggestion
		return false, nil
```

Reply:
The comment still applies, but its about not _showing_ the diff, i.e. not printing the `terraform plan` output to the user.
The caller still relies on the `hasDiff` information to create new resources (in case no state previously existed) or update them